### ISSUE TITLE
Change and more than one Independent Continuants

### ISSUE THREAD

**Issue by eliasweatherfield** [CommonCoreOntology/CommonCoreOntologies#101]
I noticed the other day that while the definition of Stasis refers to „one or more Independent Continuants“, the definition of Change speaks of „some independent continuant“. I guess the latter definition does not preclude a Change of more than one Independent Continuants outright, but still, I would submit that mentioning that possibility explicitly would be an improvement. Thanks.

**Comment by mark-jensen** [comment]
@eliasweatherfield Use of 'some' is fairly standard in many ontology definitional patterns, as it implies: "at least one, but does not preclude all". This has roots in logic, e.g, the syllogism. And I think its use in textual definitions is derived from that logically-based keyword used in existential restrictions. 

I agree consistency is a good thing to shoot for. Looking at many of the subclasses of 'stasis'. I see they also use 'some', so I would lean in favor editing `stasis` to use the venerable 'some'.

**Comment by eliasweatherfield** [comment]
I agree on consistency. However it seems to me that "one or more" is more informative because it contains information on cardinality that "some" lacks. What do you think?

**Comment by alanruttenberg** [comment]
It is not more informative than "some". If you think so, tell me something you know based on "one or more" that you don't think you can know from "some".

**Comment by eliasweatherfield** [comment]
Thank you for the inquiry. I realize I neglected to elaborate on my premisses.

(1) “some” is less informative than “one or more”/“at least one” in a definition if “some” is ambigious between “one or more”/“at least one” and “exactly one”.

(2) “some” is ambigious between “one or more“/“at least one” and “exactly one” because it is in practice used as a default existential quantifier (before a (final) decision on cardinality has been made).

Rethinking this, I have to admit that, at least off the top of my head, I can’t think of a good empirical example, at least one from a publicly released ontology, to justify (2).

**Comment by alanruttenberg** [comment]
I was at first surprised about (1) but agree there is sometimes usage that means exactly 1. "some person knocked". It would be interesting to see whether there are actual definitions that use "some" that are ambiguous. If you have a favorite please send?
But also, "some" is a term of art in OWL. The Manchester Syntax uses it specifically to mean at least 1. That said, definitions need not mirror the language used in formal assertions. In the ideal case definitions would be tested by having a variety of people reading the definition and checking whether they understood what the class is.
